libarchive/archive_windows.c

```diff
@@ -641,13 +641,14 @@ __la_waitpid(HANDLE child, int *status, int option)
 	(void)option;/* UNUSED */
 	do {
```

Comment:
Drive by question: Huh, I wonder if we should add a `WaitForSingleObject` here so it's not an expensive live wait. Perhaps later. I may also have not paged all of the code in well enough to know whether this is a correct recommendation.